macos/Sources/Features/QuickTerminal/QuickTerminalController.swift

```diff
@@ -306,6 +306,9 @@ class QuickTerminalController: BaseTerminalController {
 
     private func syncAppearance(_ config: Ghostty.Config) {
         guard let window else { return }
+        
+        // Update the quick terminal size right away
+        config.quickTerminalSize.apply(window, config.quickTerminalPosition)
```

Comment:
FYI, since you removed the function param this symbol no longer resolves and tanks the swift compilation. 

not sure if the  invocation up at 75 (72 in my rebased version) is adequate or if this wants to stay but go:

```suggestion
        
        // Update the quick terminal size right away
        self.derivedConfig.quickTerminalSize.apply(window, self.position)
```

or something else